FutureCollars/src/main/java/lesson8task1/Names.java

```diff
@@ -0,0 +1,42 @@
+package lesson8task1;
+
+import java.util.*;
+
+public class Names {
+
+    public static void main(String[] args) {
+        List<String> names = new ArrayList<>();
```

Comment:
nipotrzebnie to usuwałaś, Arrays.asList było dobrym podejściem, nie potrzebujesz tylko tego początku: new ArrayList<>(